app/wp-content/themes/kstg/library/scss/layout/_services.scss

```diff
@@ -0,0 +1,83 @@
+.services {
+  padding: 0 5% 80px;
+  background-color: $light-grey;
+  clear: both;
+  .container {
```

Comment:
@sarahlshowers: Oops, I already merged this, but you might want to go back and remove this section now that you added it to the base layout.